### ISSUE TITLE
question: is there a way to pass autolinker params?

### ISSUE THREAD

**Issue by adamgins** [copleykj/meteor-autolinker#1]
Eg something like ` {{#autolink newWindow=false }}` or add class.

The reason for asking is I am trying to work out how I can start the inappbrowser with a close button.
For this, as I understand it, I need to capture all the clicks on links to exernal content and then use something like 
```
onclick="window.open('http://apache.org', '_blank', 'location=yes')"
```
or "onclosecaption" for iOS

So I am thinking for autlinker stuff I need to add a class and look at capturing the clicks on links that way.

**Comment by shrop** [comment]
yeah, i would like to turn set `twitter` links to `false` and things like that. I don't see a way to do it currently.

**Comment by CatVas** [comment]
Hi guys! I have the same question: how to pass a "truncate" parameter? I want my links to be truncated, e. g: {{#autolink truncate=30 }}
